services/credential-server/src/apis/schema.api.ts

```diff
@@ -1,13 +1,98 @@
 import { Request, Response } from "express";
-import { ACDC_SCHEMAS } from "../utils/schemas";
+import { ResponseData } from "../types/response.type";
+import { httpResponse } from "../utils/response.util";
+import { Agent } from "../agent";
+import lmdb from "../utils/lmdb";
+import { SCHEMAS_KEY, SchemaShortDetails } from "../types/schema.type";
 
 async function schemaApi(req: Request, res: Response) {
   const { id } = req.params;
-  const data = ACDC_SCHEMAS[id];
-  if (!data) {
+  const schemas = await lmdb.get(SCHEMAS_KEY);
+  if (!schemas) {
+    return res.status(404).send("No schemas found");
+  }
+
+  const schema = schemas[id];
+  if (!schema) {
     return res.status(404).send("Schema for given SAID not found");
   }
+
+  const data = schema.schema;
+  if (!data) {
```

Comment:
Eh, we should just assume it has schema if we always store it like that. 404 implies that it's not there